frontend/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent, ReactElement } from "react";
+import cn from "../../utils/bemUtils";
+import "./Footer.scss";
+
+const Footer: FunctionComponent = (): ReactElement => {
+  const footer = cn("footer");
+
+  return (
+    <div className={footer()}>
+      <div className={footer("container")}>
+        <span className={footer("description")}>Powered by Serokell</span>
```

Comment:
I know the footer in the design says "Powered by Serokell", but people say it makes little sense:
* https://serokell.slack.com/archives/C01EASAGPKN/p1612979249086100
* https://serokell.slack.com/archives/CCK72LXUH/p1618581789010500

Looking at YouTrack as an example
![image](https://user-images.githubusercontent.com/2556409/115256408-dc866300-a137-11eb-9722-e20ad34fcfca.png)
I propose to write "Edna—the data analysis tool for researchers by Serokell". I'll ask to update the design accordingly.